libraries/bsg_manycore_request_packet.h

```diff
@@ -218,14 +219,24 @@ extern "C" {
                 packet->y_src = y;
         }
 
+        /**
+         * Set the read id in a request packet
+         * @param[in] packet a request packet
+         * @param[in] rd for for amo and int/float load
+         */
+        static inline void hb_mc_request_packet_set_reg_id(hb_mc_request_packet_t *packet, uint8_t reg_id)
```

Comment:
Why is it called "set_reg_id?"

Reply:
Previously, it was named load_id. But after the AMO change, I guess it is changed to reg_id.
I don't know why, maybe @mrutt92 can explain.